### ISSUE TITLE
Achievement activity notices being held in moderation due to too many links.

### ISSUE THREAD

**Issue by tw2113** [opencredit/BadgeOS-Community-Add-on#46]
For now, we could recommend users upping their link moderation count limit to something above 2, but we could temporarily override it in badgeos_award_achievement_bp_activity() by adding the following around the bp_activity_add() call.

```
add_filter( 'bp_bypass_check_for_moderation', '__return_true' );
add_filter( 'bp_bypass_check_for_moderation', '__return_false' );
```

**Comment by solhuebner** [comment]
Thank you for pointing this out!

**Comment by tw2113** [comment]
Fixed in 869520ef3f1a06354cd418c35ca269693ed12c85
